core/src/main/resources/application.conf

```diff
@@ -28,6 +28,11 @@ ktor {
     watch = [ classes ]
   }
 
+  cors {
+    allowedHost = "localhost:5173"
+    allowedHost = ${?WEBAPP_HOST}
```

Comment:
I would prefer to call this `UI_HOST` to use the same name as in the directory structure, and to avoid any confusion with the `WebAppReporter` which is where the name "web app" is mostly used in the context of ORT.